### ISSUE TITLE
in list 

### ISSUE THREAD

**Issue by BenjaminHarding** [Gamefroot/blockly#8]
in list 'x' get 'x' block doesn't work due to `mutationToObject` warning error. Breaks previewing.

**Comment by BenjaminHarding** [comment]
Very irritating as you cannot get a particular list item without it

**Comment by RaniSputnik** [comment]
Hey man, you know how to fix? If you go into the code for the block (not the generator) you just have to make a JSON version of their XML mutator. Sorry I won't have time today to do it myself!

**Comment by BenjaminHarding** [comment]
I'll look into it if I have a chance but I don't think I will get around to it. Too much on my plate currently

**Comment by RaniSputnik** [comment]
Fair enough. Thought I'd let you know in case it was blocking progress. Glhf!
